### DISCUSSION TITLE
Security Onion Last 2 releases Fleet/software inventorying no longer works.

### DISCUSSION THREAD

**Discussion by Hammy-72** [Security-Onion-Solutions/securityonion#10372] [Unsupported Versions]
Hi All - this is my first time joining GITHUB.  Hoping to get a little support.  I been loving this product.  However, last 2 releases I am on now "2.3.240-20230426" version wise (release date; 2023/04/26).  The Fleet option to turn on software inventorying no longer works.  It states invalid license.   Has anyone had this issue.  If I go back 3 versions it works still.   

Also - I did follow 

https://github.com/Security-Onion-Solutions/securityonion/discussions/7844#discussioncomment-2647835

It's what worked 3 versions ago.  Thanks again!

**Comment by InfosecGoon** [comment]
This appears to be a bug in FleetDM - we've opened an issue to upgrade that component in our next release, that should fix it for you.

**Reply by Hammy-72** [reply]
Great thanks!
